packages/app-graphql/README.md

```diff
@@ -5,4 +5,60 @@ title: GraphQL API
 draft: true
 [meta]-->
 
-# GraphQL API
+# GraphQL App
+
+A Keystone App that creates a GraphQL API and Apollo GraphQL playground.
+
+For information about writing queries and mutations for Keystone see the [Introduction to Keystone's GraphQL API](/guides/intro-to-keystone).
+
+## Usage
+
+```javascript
+const { Keystone } = require('@keystone-alpha/keystone');
+const { GraphQLApp } = require('@keystone-alpha/app-graphql');
+const { AdminUIApp } = require('@keystone-alpha/app-admin-ui');
+
+module.exports = {
+  new Keystone(),
+  apps: [
+    new GraphQLApp({
+        cors : { origin: true, credentials: true },
+        apiPath : '/admin/api',
+        graphiqlPath : '/admin/graphiql',
+        schemaName : 'admin',
+        apollo : {},
+    }),
+    new AdminUIApp()
+  ],
+};
+```
+
+## Config
+
+| Option         | Type     | Default                               | Description                               |
+| -------------- | -------- | ------------------------------------- | ----------------------------------------- |
+| `cors`         | `Object` | `{ origin: true, credentials: true }` | Used to enable cors                       |
+| `apiPath`      | `String` | `/admin/api`                          | Change the API path                       |
+| `graphiqlPath` | `String` | `/admin/graphiql`                     | Change the Apollo GraphQL playground path |
+| `schemaName`   | `String` | `admin`                               | Change the graphQL schema name            |
+| `apollo`       | `Object` | `{}`                                  | Options passed directly tom Apollo        |
+
+### `cors`
+
+Options passed directly to the express [cors](https://github.com/expressjs/cors) middleware package.
```

Comment:
Looks like the description is doubled up here and in the table above. Can we keep it to a single one (perhaps just the table as the descriptions are fairly short)?

Reply:
Yeah I'll review that